LICENSE

```diff
@@ -0,0 +1,35 @@
+Copyright (C) 2013-2016 Open Source Robotics Foundation, Inc; CA, USA
+Copyright (C) 2016 Janko Petereit <janko.petereit@gmail.com>
+Copyright (C) 2011-2012 Ethan Rublee <ethan.rublee@gmail.com>
```

Comment:
This kind of "aggregated" copyright statement is what we (in my case Bosch, but this applies to anybody who distributes software including this stuff) would have to put into our product documentation.

It's trivial to do this automatically when the copyright statements in the individual files are accurate. *That*, however, is the problem. I mean, for gencpp, we're not looking at a lot of changes, but this thing applies to everything in ROS.